test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -348,7 +348,7 @@ export default class BaseFormObject {
 
   getValueFromDropdown(fieldName, index = 0) {
     if (userVariables.gen3) {
-      const selectEle = this.el.find(`[data-se="${fieldName}"]`);
+      const selectEle = this.el.find(`[id="${fieldName}"]`);
```

Comment:
As part of these migrations I've been trying to fix failing tests by using alternatives to `data-se`

Reply:
Ody 1.11.1 renders `data-se` attribute for `select` (1.9.x versions were rendering `data-se` for parent `div` instead). Does it still causes test failures?